engines/postgres.py

```diff
@@ -130,5 +131,9 @@ def get_connection(self):
                              user=self.opts["user"],
                              password=self.opts["password"],
                              database=self.opts["database"])
-        conn.set_client_encoding('Latin1')
+
+        encoding = self.script.encoding
+        encoding_lookup = {'ISO-8859-1': 'LATIN1', 'utf-8': 'UTF8'}
```

Comment:
Include 'latin-1': 'LATIN1' in encoding_lookup.

Also think about a case where we have an encoding in the script like `EUC_KR`
Its not in `encoding_lookup`.